### ISSUE TITLE
Add support of accessibility name markdown in components

### ISSUE THREAD

**Issue by vvtimofeev** [gravity-ui/uikit#1562]
When we use icons as button text, screen reader applications cannot read the button's title, and our users cannot understand the meaning of the button. Therefore, we need to add support for the accessibility name property so that we can use hidden text titles for components in such cases.
<img width="1653" alt="Screenshot 2024-05-03 at 10 37 05" src="https://github.com/gravity-ui/uikit/assets/108340247/d56b0eb9-9037-4f72-abde-d6c4d7286dd6">
